web/modules/custom/search_api_field/src/Plugin/Field/FieldType/SearchItem.php

```diff
@@ -378,4 +384,22 @@ public static function storageSettingsToConfigData(array $settings) {
     return $settings;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getLimit(): int {
+    return (int) $this->get('value')->getValue()['limit'];
```

Comment:
Is the `getValue` going to always return an array with the 'limit' index in it?

Reply:
:thinking: